### PR TITLE
ROC-3567 Rename mobile to phone in code

### PR DESCRIPTION
**Pull request by filipczyks**
### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [x] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/main/features/claim/views/amount.njk [hunk 1]
```diff
@@ -20,13 +20,13 @@
         <div class="claim-amounts">
           <div id="claim-amount">
             <div>
-              <div id="claim-amount-heading-1" class="claim-amount-column-1 mobile-hide">
```
Comment (ellie-harrison) [inline src/main/features/claim/views/amount.njk]: Should this class be renamed to phone-hide (it doesn't seem related to phone numbers)?

Comment (robertparkinson) [reply src/main/features/claim/views/amount.njk]: I believe that class is to hide it on mobile devices so it doesn't show when a user is using their phone to view the site

File: src/main/features/claim/views/amount.njk [hunk 2]
```diff
@@ -20,13 +20,13 @@
...
+              <div id="claim-amount-heading-1" class="claim-amount-column-1 phone-hide">
```
Comment (robertparkinson) [inline src/main/features/claim/views/amount.njk]: this should remain as mobile-hide as it relates to hiding this when a user in on a mobile

File: src/main/features/claim/views/amount.njk [hunk 3]
```diff
@@ -20,13 +20,13 @@
...
                 <span id="claim-reason-label" class="form-label-bold">{{ t('What you’re claiming for') }}</span>
                 <span id="claim-reason-hint" class="form-hint">
                   {{ t('Briefly explain each item - for example, "broken tiles", "roof damage"') }}
                 </span>
               </div>
-              <div id="claim-amount-heading-2" class="claim-amount-column-2 mobile-hide">
+              <div id="claim-amount-heading-2" class="claim-amount-column-2 phone-hide">
```
Comment (robertparkinson) [inline src/main/features/claim/views/amount.njk]: this should remain as mobile-hide as it relates to hiding this when a user in on a mobile

File: src/main/features/claim/views/amount.njk [hunk 4]
```diff
@@ -51,12 +51,12 @@
                     </span>
                   {% endif %}
 
-                  <span class="form-label-bold claim-amount-numbered mobile-show">
+                  <span class="form-label-bold claim-amount-numbered phone-show">
```
Comment (robertparkinson) [inline src/main/features/claim/views/amount.njk]: this should remain as mobile-show as it relates to hiding this when a user in on a mobile

File: src/main/features/claim/views/amount.njk [hunk 5]
```diff
@@ -51,12 +51,12 @@
...
                     {{ t('What you’re claiming for') }}
                   </span>
 
                   {% if loop.index0 == 0 %}
-                    <span class="form-hint mobile-show">
+                    <span class="form-hint phone-show">
```
Comment (robertparkinson) [inline src/main/features/claim/views/amount.njk]: this should remain as mobile-show as it relates to hiding this when a user in on a mobile

File: src/main/features/claim/views/amount.njk [hunk 6]
```diff
@@ -68,7 +68,7 @@
                     )
                   }}
 
-                  <span class="form-label-bold mobile-show">{{ t('Amount') }}</span>
+                  <span class="form-label-bold phone-show">{{ t('Amount') }}</span>
```
Comment (robertparkinson) [inline src/main/features/claim/views/amount.njk]: this should remain as mobile-show as it relates to hiding this when a user in on a mobile

File: src/main/features/dashboard/views/index.njk [hunk 7]
```diff
@@ -23,39 +23,39 @@
           <th scope="col">{{ t('Defendant name') }}</th>
           <th scope="col">{{ t('Claim amount') }}</th>
 
-          <th scope="col" class="mobile-hide">{{ t('Status') }}</th>
+          <th scope="col" class="phone-hide">{{ t('Status') }}</th>
```
Comment (robertparkinson) [inline src/main/features/dashboard/views/index.njk]: this should remain as mobile-hide as it relates to hiding this when a user in on a mobile

File: src/main/features/dashboard/views/index.njk [hunk 8]
```diff
@@ -23,39 +23,39 @@
...
         </tr>
         </thead>
         <tbody>
         {% if claimDraftSaved %}
-          <tr class="draft mobile-table">
+          <tr class="draft phone-table">
```
Comment (robertparkinson) [inline src/main/features/dashboard/views/index.njk]: this should remain as mobile-table as it relates to how this is displayed when a user in on a mobile and the other places in this file

File: src/main/features/dashboard/views/macro/caseStatus.njk [hunk 9]
```diff
@@ -38,7 +38,7 @@
     { responseDeadline: claim.responseDeadline | date }) }}
   {% elseif status === ClaimStatus.NO_RESPONSE %}
     <a href="{{ url }}">{{ t('Respond to claim') }}</a>
-    <span class="block mobile-inline">
+    <span class="block phone-inline">
```
Comment (robertparkinson) [inline src/main/features/dashboard/views/macro/caseStatus.njk]: as above

File: src/main/features/first-contact/views/start.njk [hunk 10]
```diff
@@ -22,7 +22,7 @@
 
       <p>{{ t('They can use a CCJ to get the money from you, for example by applying to the court to send bailiffs to your home or business.') }}</p>
 
-      <p>{{ t('A CCJ may make it harder to get credit, such as a mobile phone contract, credit card or mortgage.') }}</p>
+      <p>{{ t('A CCJ may make it harder to get credit, such as a phone contract, credit card or mortgage.') }}</p>
```
Comment (robertparkinson) [inline src/main/features/first-contact/views/start.njk]: this is content and shouldn't be changed. should only be changing variable, paths and file names

File: src/main/features/response/form/models/statement-of-means/monthlyExpenses.ts [hunk 11]
```diff
@@ -50,9 +50,9 @@ export class ValidationErrors {
   static readonly AMOUNT_INVALID_DECIMALS_TV: string = `${GlobalValidationErrors.AMOUNT_INVALID_DECIMALS} for TV and Broadband`
   static readonly NON_NEGATIVE_NUMBER_REQUIRED_TV: string = `${GlobalValidationErrors.NON_NEGATIVE_NUMBER_REQUIRED} for TV and Broadband`
 
-  static readonly AMOUNT_REQUIRED_MOBILE: string = `${GlobalValidationErrors.AMOUNT_REQUIRED} for Mobile Phone`
-  static readonly AMOUNT_INVALID_DECIMALS_MOBILE: string = `${GlobalValidationErrors.AMOUNT_INVALID_DECIMALS} for Mobile Phone`
-  static readonly NON_NEGATIVE_NUMBER_REQUIRED_MOBILE: string = `${GlobalValidationErrors.NON_NEGATIVE_NUMBER_REQUIRED} for Mobile Phone`
+  static readonly AMOUNT_REQUIRED_MOBILE: string = `${GlobalValidationErrors.AMOUNT_REQUIRED} for Phone`
```
Comment (robertparkinson) [inline src/main/features/response/form/models/statement-of-means/monthlyExpenses.ts]: Can you change the end of the variable name here for these three please?

File: src/main/features/response/views/statement-of-means/bank-accounts.njk [hunk 12]
```diff
@@ -26,7 +26,7 @@
     {% if isJointError %}<span id="{{ isJointKey }}[label]" class="error-message">{{ t(isJointError) }}</span>{% endif %}
     {% if balanceError %}<span id="{{ balanceKey }}[label]" class="error-message">{{ t(balanceError) }}</span>{% endif %}
 
-    <label class="form-label-bold mobile-show" for="{{ typeOfAccountKey }}">{{ index + 1 }} {{ t('. Type of account') }}</label>
+    <label class="form-label-bold phone-show" for="{{ typeOfAccountKey }}">{{ index + 1 }} {{ t('. Type of account') }}</label>
```
Comment (robertparkinson) [inline src/main/features/response/views/statement-of-means/bank-accounts.njk]: as above this shouldn't be changing and the other places in this file

File: src/main/features/response/views/statement-of-means/debts.njk [hunk 13]
```diff
@@ -27,7 +27,7 @@
     {% if totalOwedError %}<span id="{{ totalOwedKey }}[label]" class="error-message">{{ t(totalOwedError) }}</span>{% endif %}
     {% if monthlyPaymentsError %}<span id="{{ monthlyPaymentsKey }}[label]" class="error-message">{{ t(monthlyPaymentsError) }}</span>{% endif %}
 
-    <label class="form-label-bold mobile-show" for="{{ debtKey }}">{{ index + 1 }} {{ t('. List each debt') }}</label>
+    <label class="form-label-bold phone-show" for="{{ debtKey }}">{{ index + 1 }} {{ t('. List each debt') }}</label>
```
Comment (robertparkinson) [inline src/main/features/response/views/statement-of-means/debts.njk]: as above, this shouldn't be changing and the other places in this file

File: src/main/features/response/views/statement-of-means/debts.njk [hunk 14]
```diff
@@ -64,14 +64,14 @@
             {% if noRowsError %}<span class="error-message" id="rows[label]">{{ t(noRowsError) }}</span>{% endif %}
 
             <div class="multiline-rows">
-              <div class="triple-column mobile-hide">
-                <div class="column-1 mobile-hide">
+              <div class="triple-column phone-hide">
```
Comment (robertparkinson) [inline src/main/features/response/views/statement-of-means/debts.njk]: as above, this shouldn't be changing and the other places in this file

File: src/main/features/response/views/statement-of-means/monthly-expenses.njk [hunk 15]
```diff
@@ -22,7 +22,7 @@
     {% if amountError %}<span id="{{ amountKey }}[label]" class="error-message">{{ t(amountError) }}</span>{% endif %}
     {% if descriptionError %}<span id="{{ descriptionKey }}[label]" class="error-message">{{ t(descriptionError) }}</span>{% endif %}
 
-    <label class="form-label-bold mobile-show" for="{{ descriptionKey }}">{{ index + 1 }} {{ t('. Name of expense') }}</label>
+    <label class="form-label-bold phone-show" for="{{ descriptionKey }}">{{ index + 1 }} {{ t('. Name of expense') }}</label>
```
Comment (robertparkinson) [inline src/main/features/response/views/statement-of-means/monthly-expenses.njk]: as above, this shouldn't be changing and the other places in this file

File: src/main/features/response/views/statement-of-means/monthly-income.njk [hunk 16]
```diff
@@ -22,7 +22,7 @@
     {% if amountError %}<span id="{{ amountKey }}[label]" class="error-message">{{ t(amountError) }}</span>{% endif %}
     {% if descriptionError %}<span id="{{ descriptionKey }}[label]" class="error-message">{{ t(descriptionError) }}</span>{% endif %}
 
-    <label class="form-label-bold mobile-show" for="{{ amountKey }}">{{ index + 1 }} {{ t('. Name of income') }}</label>
+    <label class="form-label-bold phone-show" for="{{ amountKey }}">{{ index + 1 }} {{ t('. Name of income') }}</label>
```
Comment (robertparkinson) [inline src/main/features/response/views/statement-of-means/monthly-income.njk]: as above, this shouldn't be changing and the other places in this file

File: src/main/public/stylesheets/application.scss [hunk 17]
```diff
@@ -22,7 +22,7 @@ $path: '/img/lib/';
 @import 'components/report-problem';
 @import 'components/dashboard';
 @import 'components/survey-links';
-@import 'components/mobile';
+@import 'components/phone';
```
Comment (robertparkinson) [inline src/main/public/stylesheets/application.scss]: this shouldn't be changing as the file relates to how things are displayed on mobiles rather than asking about a phone number

File: src/main/public/stylesheets/components/_phone.scss [hunk 18]
```diff
@@ -1,28 +1,28 @@
-.mobile-show,
-.mobile-table-status {
+.phone-show,
```
Comment (robertparkinson) [inline src/main/public/stylesheets/components/_phone.scss]: as above, this shouldn't be changing as it relates to how things are displayed on a mobile and the other places in this file

File: src/main/views/macro/timelineFragment.njk [hunk 19]
```diff
@@ -22,15 +22,15 @@
     </span>
     {% endif %}
 
-    <label class="form-label-bold mobile-show" for="{{ dateKey }}">{{ index + 1 }}{{ t('. Date') }}</label>
-    <span class="form-hint mobile-show">{{ t('For example, 1 May 2017') }}</span>
+    <label class="form-label-bold phone-show" for="{{ dateKey }}">{{ index + 1 }}{{ t('. Date') }}</label>
```
Comment (robertparkinson) [inline src/main/views/macro/timelineFragment.njk]: as above, this shouldn't be changing and the other places in this file

**Comment by filipczyks** [comment]
All comments from code review were adressed, thanks

File: src/integration-test/tests/citizen/claim/pages/citizen-phone.ts [hunk 20]
```diff
@@ -0,0 +1,23 @@
...
+  submit: 'input[type=submit]'
+}
+
+export class CitizenPhonePage {
+
+  open (type: string): void {
+    I.amOnCitizenAppPage(`/claim/${type}-phone`)
+  }
+
+  enterPhone (phoneNumber: string): void {
```
Comment (yarpo) [inline src/integration-test/tests/citizen/claim/pages/citizen-phone.ts]: `enterPhoneNumber`?

#suggestion

File: src/integration-test/tests/citizen/defence/pages/defendant-phone.ts [hunk 21]
```diff
@@ -0,0 +1,19 @@
...
+  phoneNumber: 'input[id=number]'
+}
+
+const buttons = {
+  submit: 'input[type=submit]'
+}
+
+export class DefendantPhonePage {
+
+  enterPhone (phoneNumber: string): void {
```
Comment (yarpo) [inline src/integration-test/tests/citizen/defence/pages/defendant-phone.ts]: as above

File: src/main/features/response/views/statement-of-means/check-and-send.njk [hunk 22]
```diff
@@ -141,7 +141,7 @@
 {{ row('School costs (include clothing)', expenses.schoolCosts | numeral, '', false) }}
 {{ row('Food and Housekeeping', expenses.foodAndHousekeeping | numeral, '', false) }}
 {{ row('TV and Broadband', expenses.tvAndBroadband | numeral, '', false) }}
-{{ row('Mobile phone', expenses.mobilePhone | numeral, '', false) }}
+{{ row('Phone', expenses.phone | numeral, '', false) }}
```
Comment (yarpo) [inline src/main/features/response/views/statement-of-means/check-and-send.njk]: Are you sure this should also be updated?

It's a monthly expense on phone, not phone number. Please ask somebody from UX team

#REQUEST.

Comment (yarpo) [reply src/main/features/response/views/statement-of-means/check-and-send.njk]: I had to add this comment to be able to submit my review with request

**Comment by geek-soft** [comment]
Closing for now due to lack of activity.
